### ISSUE TITLE
TransactionalConnection in plugins failing with not found, or not in "default" connection

### ISSUE THREAD

**Issue by chladog** [vendure-ecommerce/vendure#504]
**Describe the bug**
After updating to new transactional connection I get errors in plugins everywhere the connection is being used:
__No repository for "[object Object]" was found. Looks like this entity is not registered in current "default" connection._
The repository I'm trying to access is of custom entity. 
With trial of latest version of ExamplePlugin(https://github.com/vendure-ecommerce/plugin-template) I get similiar error: 
_UnhandledPromiseRejectionWarning: ConnectionNotFoundError: Connection "default" was not found._

**Expected behavior**
Connection working as before

**Environment (please complete the following information):**
 - @vendure/core version: v16.0.0
 - Nodejs version
 - Database (mysql/postgres etc): postgres

After reverting back to InjectedConnection plugins started working as before again.

**Comment by michaelbromley** [comment]
I just tested this in the `plugin-template` repo (latest master branch) and I initially got some "module not found" errors. I fixed this by re-building the docker image (`yarn dev:rebuild`) and after that I ran `yarn dev:run` which works without errors.

One thing that is worth trying is to delete your lockfile (and potentially node_modules) and reinstall. I found that on upgrading there was some unnecessary duplication of dependencies so had to delete my lockfile. Not sure if that will fix it but, since I cannot reproduce this issue either in my own upgraded projects (with many custom entities) nor the plugin-template, I suspect there may be some issue related to your local environment.
